freq/views.py

```diff
@@ -440,7 +444,10 @@ def get(self, request, *args, **kwargs):
 
 
 class BaseView(BaseViewMixin, FreqTemplateView):
-    pass
+
+    @property
+    def timeseries_selection(self):
+        return self.request.session['startpage']['timeseries_selection']
```

Comment:
If one of the two keys isn't available, you get a nasty error.
Perhaps a more forgiving `.get()`? Perhaps even one that sets a default perhaps? 
And perhaps split it into two lines so that you know from the error which of the two failed.